Mathlib/MeasureTheory/Function/LocallyIntegrable.lean

```diff
@@ -26,9 +26,11 @@ open MeasureTheory MeasureTheory.Measure Set Function TopologicalSpace Bornology
 
 open scoped Topology Interval ENNReal
 
-variable {X Y E F R : Type*} [MeasurableSpace X] [TopologicalSpace X]
+variable {X Y ε ε' ε'' E F R : Type*} [MeasurableSpace X] [TopologicalSpace X]
```

Comment:
That's a really weird name for a space: why `ε`?

Reply:
Yes, but what if you want to introduce an arbitrary `ε > 0` 😱

Reply:
Hm, I can try.

Reply:
In general, once a file is generalised, I'm happy to rename everything back to E. That's just not fun in the middle of a proof.